test/verify/check-system-info

```diff
@@ -34,6 +34,15 @@ VERSION_ID="2.0"
 PRETTY_NAME="Foobar Adventure Linux Server 2.0 (Day of Doom)"
 """
 
+# Setting the locale via env results in warnings and breaks the threads/cpu
+# detection, so use python
+lscpu = """#!/usr/bin/python{1}
```

Comment:
Why does this need to use Python as opposed to simple shell? Do you actually *want* the warnings about wrong locale? (They should go to stderr, so the parsing wouldn't see them either way)

Reply:
using sh results in the script just exiting with an error on platforms where setting locale via env isn't supported.